app/src/main/java/chat/rocket/android/authentication/login/ui/LoginFragment.kt

```diff
@@ -58,18 +72,27 @@ class LoginFragment : Fragment(), LoginView {
         }
     }
 
+    override fun onConnected(p0: Bundle?) {
```

Comment:
Please, use a meaningful name here. What is `p0`? Is it a Bundle? So use `bundle: Bundle`. Also, check if it works when using Bundle instead of Bundle? (sometimes the Kotlin plugin doesn't know about the nullability and tends to declare a parameter as nullable).